common/src/app/common/files/repair.cljc

```diff
@@ -68,7 +68,19 @@
 
 (defmethod repair-error :child-not-found
   [_ {:keys [shape page-id args] :as error} file-data _]
-  (log/info :hint "Repairing shape :child-not-found" :id (:id shape) :name (:name shape) :page-id page-id)
+  (let [repair-shape
+        (fn [parent-shape]
+          (log/debug :hint "  -> Remove children from" :shapes (:child-id args))
```

Comment:
I think it would be clearer `(log/debug :hint "  -> Remove child" :child-id (:child-id args))`